order-book/src/main/java/com/market/banica/order/book/helper/ConstantsAuroraRequestHandler.java

```diff
@@ -0,0 +1,24 @@
+package com.market.banica.order.book.helper;
+
+import java.util.regex.Pattern;
+
+public class ConstantsAuroraRequestHandler {
```

Comment:
the pattern for naming class with constants is Constants to be at the end - like for the tests. So better naming here would be the module name + Constants (this is because the class could be used across the whole module and should be common for all classes)

Reply:
Fixed